compiler/optimizer/Inliner.cpp

```diff
@@ -1478,10 +1478,10 @@ TR_InlinerBase::createVirtualGuard(
          break;
       }
    TR::DebugCounter::prependDebugCounter(comp(),
-      TR::DebugCounter::debugCounterName(comp(), "virtualGuards.byKind/%s/(%s)/bcinfo=%d.%d", tracer()->getGuardKindString(guard), comp()->signature(), bcInfo.getCallerIndex(), bcInfo.getByteCodeIndex()),
+      TR::DebugCounter::debugCounterName(comp(), "virtualGuards.byKind/%s/(%s)/bcinfo=%d.%d/fidelity=%d", tracer()->getGuardKindString(guard), comp()->signature(), bcInfo.getCallerIndex(), bcInfo.getByteCodeIndex(), fidelity),
```

Comment:
I agree - I don't see why the fidelity needs to be in the counter - it doesn't help you understand the virtualGuard behaviour that we are trying to count.